util/screenshots.js

```diff
@@ -0,0 +1,69 @@
+import fs from "fs";
+import path from "path";
+import * as warcio from "warcio";
+
+
+// ============================================================================
+
+function isScreenshotSelected(params) {
+  return params.screenshot || params.fullPageScreenshot || params.thumbnail;
+}
+
+class Screenshots {
+
+  constructor({page, id, url, date, directory}) {
+    this.page = page;
+    this.id = id;
+    this.url = url;
+    this.directory = directory;
+    this.warcName = path.join(this.directory, "screenshots-" + this.id + ".warc.gz");
```

Comment:
- We should use one WARC file per browsertrix-crawler instance, so that crawls with 1000s of pages don't result in thousands of WARC files.
- Probably should just place the WARC files in the archives/ dir, but with the `screenshots-` prefix, so that they are included in the WACZ

Reply:
I have the screenshots now being written to `archives/screenshots.warc.gz` and removed the extra logic that was previously adding warcs in the `snapshots` dir to the WACZ